numpy/random/_generator.pyi

```diff
@@ -0,0 +1,245 @@
+import sys
+from typing import Any, Callable, Dict, Optional, Sequence, Tuple, Union, overload
+
+from numpy import dtype, float32, float64, int64, integer, ndarray
+from numpy.random import BitGenerator
+from numpy.typing import ArrayLike, DTypeLike, _ArrayLikeFloat_co, _ArrayLikeInt_co, _ShapeLike, _DoubleCodes, _SingleCodes
+
+if sys.version_info >= (3, 8):
+    from typing import Literal
+else:
+    from typing_extensions import Literal
+
+class Generator:
+    # COMPLETE
+    _bit_generator: BitGenerator
+    _poisson_lam_max: float
```

Comment:
The `_bit_generator` one is kind of important (to me at least) in case you forget to grab it somewhere else.  That is was left public (in Cython) but marked private was a compromise.

Reply:
Isn't `_bit_generator` already publicly accessible via the `bit_generator` property though?

Reply:
You are right.  I've removed it.